src/gettext.cpp

```diff
@@ -466,12 +461,8 @@ inline const char* is_unlocalized_string2(const std::string& str, const char* si
 
 std::string dsngettext (const char * domainname, const char *singular, const char *plural, int n)
 {
-	std::string msgval;
-	//Braces so that only the line that needs to be in the lock is in it:
-	{
-		std::scoped_lock lock(get_mutex());
-		msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
-	}
+	std::string msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
+	
```

Comment:
One of the things that the CI checks is that there are no lines containing only whitespace or has trailing whitespace, so you need to delete the tab here.